### DISCUSSION TITLE
HTTP Error: 400 Authentication failed

### DISCUSSION THREAD

**Discussion by Rand0x** [scnr/installer#30] [Q&A]
I am running a security scan (SCNR) using the WebUI, and I use a Ruby script for logging in. I have successfully tested the Ruby script for logging in to the website with no issues.

However, when I paste the same script into the WebUI, it correctly populates all the login values (I've verified this with printouts). But after clicking the login button, the website throws an error, causing the scan to stop.

How can I ignore this error and allow the scan to continue? After every login an error causes (401 HEAD).

![asdf](https://github.com/scnr/installer/assets/81554629/30f7e144-6deb-4634-b8a5-642def0e8f4f)

**Comment by Zapotek** [comment]
Hello,

I'm going to need access to both the script and the website to debug this, would that be possible?
You can share privately of course.

Cheers
